libmogwai-schedule/connection-monitor-nm.c

```diff
@@ -99,6 +99,23 @@ static gboolean             mws_connection_monitor_nm_get_connection_details (Mw
  * calculated as the pessimisic combination of the metered status of each
  * #NMDevice and #NMSettingConnection associated with that active connection.
  *
+ * Several settings from #NMSettingUser are read for each active connection:
+ *
+ *  * `connection.allow-downloads` (boolean): If `1`, big downloads are allowed
+ *    on this connection. If `0`, they are not, and Mogwai will never schedule
+ *    downloads on this connection. (Default: `1`.)
```

Comment:
I'd make it more detailed as `Mogwai will never schedule downloads on this connection` may be confused by "Mogwai will never schedule my entries?". Something like:
```
If `0`, they are not, and Mogwai will never never notify the scheduled entries to perform downloads (Default: `1`.)
```

Reply:
I’m not sure what you mean by “may be confused by "Mogwai will never schedule my entries?"”. Your suggested text is not quite correct, since a schedule entry may be scheduled for download on a *different* NM connection — note that `connection.allow-downloads` is a per-connection setting. The user may have multiple active connections.

However, at the moment I’m assuming that no EOS users actually do have multiple simultaneously active connections (is that a fair assumption?), since Mogwai doesn’t currently support indicating *which* connections to download over (essentially, which network interfaces to use or avoid). Support for that will be added in future.

Reply:
I was referring that it sounds like entries don't get added to the scheduler but they do. In any case, I think the clarification you wrote does the job.